code/style.css

```diff
@@ -1,5 +1,125 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
-
 body {
-  background-color: lightblue;
+  font-family: 'Roboto', Verdana, Calibri, sans-serif;
+  font-size: 14px;
+  margin: 0;
+}
+
+h1, h2 {
+  font-family: 'Segoe UI', Verdana, Calibri, sans-serif;
+  font-size: 50px;
+  text-align: center;
+  margin: 0;
+}
+
+a.black:link, a.black:visited {
+  color: black;
+  text-decoration: underline;
+}
+a.white:link, a.white:visited {
+  color: whitesmoke;
+  text-decoration: none;
+}
+
+#titel {
+  background-image: url(./images/gruppe.jpeg);
+  background-size: cover;
+  height: 600px;
+}
+h1 {
+  color: rgb(22, 22, 76);
+  padding: 15px;
+}
+
+header {
+  background-color: rgb(38, 62, 168);
+  padding: 10px;
+}
+h2 {
+  color: whitesmoke;
+}
+
+nav {
+  padding-top: 20px;
+  padding-right: 40px;
```

Comment:
There's a shorthand allowing to right it on one line:
`padding: 20px 40px 0 0;` (the four values are from top to left in clockwise order)